t/t-install.sh

```diff
@@ -224,6 +225,34 @@ begin_test "install --local"
 )
 end_test
 
+begin_test "install --local with failed permissions"
```

Comment:
What causes the install to fail in this case?

Reply:
The `chmod 500 .git` operation. The directory isn't writable, so `.git/config` can't be updated.

Let me add a comment to that effect.